docs/open-source-project/release-notes/tizen-5-0-m1.md

```diff
@@ -0,0 +1,952 @@
+
+​		**o**   **Support for kernel level audit has been added**
+
+​		**o**   **Major protection profiles(CAPP, LSPP, PCI-DSS, NISPOM, STIG) have been added**
+
+​	**·**        **Security Analyzer**
+
+​		**o**   **Provide general information of Tizen security**
+
+​		**o**   **Provide access control information of running processes**
+
+​		**o**   **Check security related logs**
+
+​		**o**   **Provide basic guides for developers**
+
+ 
+
+## **Service Framework**
+
+**New and Changed Features**
+
+​	**·**        **Account**
+
+​		**o**   **Account module is added in TV profile.**
+
+**Fixes**
+
+​	**·**        **Phonenumber-util**
+
+​		**o**   **Fixed D-Bus policies**
+
+​		**o**   **Fixed memory leaks & potential defects**
+
+​	**·**        **Account-Manager**
+
+​		**o**   **Fixed D-Bus Policies**
+
+​		**o**   **Fixed Memory leaks & potential defects**
+
+​	**·**        **Sync-manager**
+
+​		**o**   **Fixed D-bus policies**
+
+​		**o**   **Fixed memory leak & potential defects**
+
+​	**·**        **Geofence-server**
+
+​		**o**   **Fixed D-bus policies**
+
+​		**o**   **Fixed memory leak & potential defects**
+
+​	**·**        **LBS Server**
+
+​		**o**   **Fixed D-bus policies**
+
+​		**o**   **Fixed memory leak & potential defects**
+
+​	**·**        **Maps-plugin-here**
+
+​		**o**   **Fixed D-bus policies**
+
+​		**o**   **Fixed memory leak & potential defects.**
+
+​	**·**        **Sensor**
+
+​		**o**   **Fixed Memory leaks & potenatil defects**
+
+​	**·**        **Email**
+
+​		**o**   **Fixed memory leak & potential defects**
+
+​	**·**        **Message-Service**
+
+​		**o**   **Fixed memory leaks & potential defects**
+
+​	**·**        **Context**
+
+​		**o**   **Fixed memory leaks & potential defects**
+
+​	**·**        **Calendar**
+
+​		**o**   **Fixed memory leaks & potential defects**
+
+​	**·**        **Location** 
+
+​		**o**   **Fixed memory leaks & potential defects**
+
+##  
+
+## **Web Framework**
+
+ **New and Changed Features**
+
+​	**·**        **Web Engine**
+
+​		**o**   **Applying latest open source chromium(blink) : from M56 to M63**
+
+​			**§**  **Provide new W3C Standard  (Animated PNG, Full featured Indexed DB, CSS grid layout, Web USB, MSE/EME)**
+
+​			**§**  **Enhancement standard web features (WebRTC, ARIA 1.1)**
+
+​			**§**  **Web performance enhancement (V8 : ES 6 performance enhancement, Web Assembly default enabling)**
+
+ 
+
+## **Tizen .NET**
+
+ **New and Changed Features**
+
+​	**·**        **.NET Runtime**
+
+​		**o**   **Memory optmization patches have been merged.**
+
+​	**·**        **.NET Launcher**
+
+​		**o**   **Plugin interface for dll preloading is added.**
+
+​	**·**        **Xamarin.Forms**
+
+​		**o**   **Latest Xamarin.Forms stable version (3.0.0) support.**
+
+​		**o**   **Xamarin.Forms nuget package (Xamarin.Forms.nupkg) supports tizen as a default platform since Xamarin.Forms 3.0.0.**
+
+​		**o**   **TargetIdiom.Watch is now available for wearable profile.**

```

Comment:
TargetIdiom.Watch is now available for wearable profile since Xamarin.Forms 3.1 Pre Release 1.2